README.md

```diff
@@ -27,11 +26,20 @@ The secrets directory is **not** copied to `test` and `live` during deployments
 You can create all your keys in the live environment and then Clone files to other environments to copy the keys.
 
 ## Installation
-For help installing, see [Manage Plugins](https://pantheon.io/docs/terminus/plugins/)
+
+### Installing Secrets 3.x:
```

Comment:
Only the Build Tools plugin has different major versions that align to different Terminus versions. Other plugins have a single version that may be installed on Terminus 3 or Terminus 2.

Reply:
Would `Installing Secrets in Terminus 3.x` be better?

Reply:
Yeah; see README of the Terminus plugin example, and follow that format.

Reply:
Updated.